src/ray/core_worker/reference_count.h

```diff
@@ -313,6 +355,10 @@ class ReferenceCounter {
     /// task's caller is also a borrower. The key is the task's return ID, and
     /// the value is the task ID and address of the task's caller.
     absl::flat_hash_map<ObjectID, rpc::WorkerAddress> stored_in_objects;
+    /// The number of tasks that depend on this object that may be retried in
+    /// the future (pending execution or finished but retryable). If the object
+    /// is direct (not stored in plasma), then its lineage ref count is 0.
```

Comment:
add " ... because the object is inlined and we don't need to track it"?

Alternatively, consider tracking it anyways and making the eviction decision based on the store state.